### ISSUE TITLE
Scheduled task failed  to perform 

### ISSUE THREAD

**Issue by rmkhalfay** [d4rken-org/sdmaid-se#1316]
I have password set for my device. Accessibility access is granted but yet the app says failure as screen not accessible and does not conduct scheduled task. Even if Shizuku is installed same thing happens.

**Comment by MarkyMarkDE** [comment]
The Display must active to or SDMaid SE does fail

**Comment by rmkhalfay** [comment]
SD Maid Pro works this way

**Comment by d4rken** [comment]
@MarkyMarkDE is correct.

> SD Maid Pro works this way

SD Maid 1 had the same restrictions, it may only have looked different to you. SD Maid 1 retries for many more minutes so if SD Maid 1 tried, but the screen was locked, then SD Maid 1 may have retried for up to an hour. If you then later unlock the screen, it could look like SD Maid 1 was working correctly the whole time. It's not possible for an accessibility service to "click" something behind the lock screen. It can only click what you can click.

**Comment by rmkhalfay** [comment]
Thank you. I have removed the schedule tasks from my settings. As I cannot keep the phone unlocked without password due to office email security requirements.
